src/sparseml/recipe_template/utils.py

```diff
@@ -142,3 +150,110 @@ def modifier_recipe_variables(self) -> Dict[str, Any]:
         },
     ),
 }
+
+
+# getters
+
+
+def get_quantization_info(
+    quantization: Union[str, bool] = False,
+    target: str = "vnni",
```

Comment:
shall we add checks to make sure that the argument for `target` is one of the [vnni, tensorrt etc]?